app/components/modals/invite-new-user.js

```diff
@@ -2,6 +2,7 @@ import RSVP from 'rsvp';
 import injectService from 'ember-service/inject';
 import {A as emberA} from 'ember-array/utils';
 import run from 'ember-runloop';
+import $ from 'jquery';
```

Comment:
That actually gives a `Cannot read property '$' of undefined` because `this` is undefined (b/c it is run later im guessing).